back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/voting_results.rb

```diff
@@ -68,12 +68,12 @@ def self.trigger_multiloc_key
     end
 
     def generate_commands(recipient:, activity:)
-      basket = activity.item
+      notification = activity.item
       [{
         event_payload: {
-          project_url: Frontend::UrlService.new.model_to_url(project, locale: Locale.new(recipient.locale)),
```

Comment:
"project" on this line was causing my test to fail, since it doesn't exist as a local variable :thinking: I changed this (and basket since we didn't need both) just to use the notification project/phase. Hopefully that's ok?